### PR TITLE
Add Series.iteritems() and Series.items() (alias)

### PR DESCRIPTION
**Pull request by itholic**
This PR proposes `Series.iteritems`.

```python
>>> s = ks.Series(['A', 'B', 'C'])
>>> for index, value in s.items():
...     print(f"Index : {index}, Value : {value}")
Index : 0, Value : A
Index : 1, Value : B
Index : 2, Value : C
```

### CODE REVIEW

File: databricks/koalas/series.py [hunk 1]
```diff
@@ -4975,6 +4975,49 @@ def item(self):
...
+        >>> for index, value in items:
+        ...     print("Index : {}, Value : {}".format(index, value))
+        Index : 0, Value : 0
+        Index : 1, Value : 1
+        Index : 2, Value : 2
+        Index : 3, Value : 3
+        Index : 4, Value : 4
+        """
+        kser = self[: ks.options.compute.max_rows]
+        return zip(iter(kser.index.to_numpy()), kser.values)
```
Comment (HyukjinKwon) [inline databricks/koalas/series.py:5015]: Can you follow the implementation of `DataFrame.iterrows`?

Comment (itholic) [reply databricks/koalas/series.py:5015]: @HyukjinKwon 

Oops, sorry for the check late.

I'll manage this. Thanks!!

**Comment by ueshin** [comment]
@itholic Could you update the description? Now that we won't limit the row number by `compute.max_rows`, right?

**Comment by itholic** [comment]
@ueshin You're right. I just managed them, thanks! :)

**Pull request merged**